specs/WinRT/WinRTAPIContracts.md

```diff
@@ -0,0 +1,78 @@
+
+
+
+# Windows App SDK API Contracts
+
+## Summary
+Customers trust that Microsoft products are sound.  As Windows App SDK evolves and more developers use Windows App SDK in their own projects Windows App SDK needs a way, at build time and run time, to allow apps to light up API features and to gracefully degrade.  This can be achieved by looking at the deltas between APIs.  
```

Comment:
Don't make markdown lines >100 chars long

https://github.com/microsoft/WindowsAppSDK/blob/main/docs/Coding-Guidelines/Languages-Markdown.md

VSCode has some nice extensions that automagically format paragraphs or do so at a keystroke (alt-q seems popular).